python/paddle/fluid/initializer.py

```diff
@@ -17,8 +17,7 @@
 import contextlib
 
 __all__ = [
-    'Constant', 'Uniform', 'Normal', 'Xavier', 'force_init_on_cpu',
-    'init_on_cpu'
+    'ConstantInitializer', 'UniformInitializer', 'NormalInitializer', 'XavierInitializer', 'Constant', 'Uniform', 'Normal', 'Xavier', 'force_init_on_cpu','init_on_cpu'
```

Comment:
I am not sure why we need to do this? Constant == ConstantInitializer, Uniform = UniformInitializer and so on. Could you elaborate why we need this change

Reply:
https://github.com/PaddlePaddle/Paddle/issues/9761
The above issue mentions the display error caused by operations like ```Constant==ConstantInitializer```, and the documents are all auto generated by [gen_doc.py](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/api/gen_doc.py), which traverses items in ```__all__```. I've asked @reyoung for suggestions and decide to solve this problem by add suffixes to alias name. e.g ```Constant -> ConstantInitializer``` and expose both of them.